LBHackney-IT/lbh-social-care
# Add API endpoints

**What**  
Added API endpoint integration for _cases_ and _residents_.

**Extras**
- renamed tests in specs
- added .env to jest config
- added .env.sample


**How to test it**
- checkout this branch
- set the env variables `AWS_AUTHORIZATION`, `AWS_KEY`, `ENDPOINT_RESIDENTS`, `ENDPOINT_CASES` (ping me if you need them :))
- install the new dependencies
- spin up the app
- check that the APIs are working
  - `/api/cases/33317112`
  - `/api/residents/4`
  - `/api/residents?last_name=scott`
